### PR TITLE
Update dns.md

### PR DESCRIPTION
**Pull request by sirdarckcat**

### CODE REVIEW

File: docs/dns.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # DNS Support
-If you point a domain or subdomain to Google Cloud DNS, you can use `kctf-setup-dns` and `kctf-chal-dns` to map *[challenge_name].yourdomain.com* to your challenge.
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF will create a DNS zone that maps the challenge to your challenge's IP address.
 
 Follow these steps:
 
 1. Configure the subdomain in `kctf-setup-config-create`.
```
Comment (sroettger) [inline docs/dns.md]: name the flag here? (--domain-name)

File: docs/dns.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
 
 Follow these steps:
 
 1. Configure the subdomain in `kctf-setup-config-create`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges will be deployed on *xxxxx.foo.example.com*.
+1. Run `kctf-cluster-start`
 1. Point the subdomain to Google Cloud DNS. This requires [adding an NS record on your name server](https://cloud.google.com/dns/docs/update-name-servers).
```
Comment (sroettger) [inline docs/dns.md:11]: should this add a step about setting dns=true in the chal config?

File: docs/dns.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
 # DNS Support
-If you point a domain or subdomain to Google Cloud DNS, you can use `kctf-setup-dns` and `kctf-chal-dns` to map *[challenge_name].yourdomain.com* to your challenge.
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF will create a DNS zone that maps the challenge to your challenge's IP address.
```
Comment (goedi02) [inline docs/dns.md]: cosmetic nit, you can simplify by removing the future tense without changing the meaning: "...kCTF creates a DNS zone..."

File: docs/dns.md [hunk 4]
```diff
@@ -1,12 +1,10 @@
...
 
 Follow these steps:
 
 1. Configure the subdomain in `kctf-setup-config-create`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges will be deployed on *xxxxx.foo.example.com*.
```
Comment (goedi02) [inline docs/dns.md]: cosmetic nit, you can simplify by removing the future tense without changing the meaning: "...the challenges are deployed on..."

**Review by goedi02 — COMMENTED** [review]
Added two comments with small cosmetic nits.

**Comment by sirdarckcat** [comment]
done :)

File: docs/dns.md [hunk 5]
```diff
@@ -1,12 +1,11 @@
...
 Follow these steps:
 
-1. Configure the subdomain in `kctf-setup-config-create`. 
+1. Configure the domain in `kctf-setup-config-create --domain-name foo.example.com`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges are be deployed on *xxxxx.foo.example.com*.
+1. Run `kctf-cluster-start`
+1. Add `DNS=true` on `config/challenge.conf`.
```
Comment (sroettger) [inline docs/dns.md:10]: this is now in challenge.yaml

File: docs/dns.md [hunk 6]
```diff
@@ -1,12 +1,11 @@
...
 1. Point the subdomain to Google Cloud DNS. This requires [adding an NS record on your name server](https://cloud.google.com/dns/docs/update-name-servers).
```
Comment (sroettger) [inline docs/dns.md:11]: That page tells you to run `gcloud dns managed-zones describe $ZONE`, but it's clear what ZONE to use.
I.e. I found it with `gcloud dns managed-zones list` IIRC.

File: docs/dns.md [hunk 7]
```diff
@@ -1,12 +1,11 @@
...
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF creates a DNS zone that maps the challenge to your challenge's IP address.
 
 Follow these steps:
 
-1. Configure the subdomain in `kctf-setup-config-create`. 
+1. Configure the domain in `kctf-setup-config-create --domain-name foo.example.com`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges are be deployed on *xxxxx.foo.example.com*.
```
Comment (sroettger) [inline docs/dns.md:8]: unless if it's a web challenge, then it's at www.xxxxx.foo.example.com

File: docs/dns.md (hunk 5 above)
Comment (sirdarckcat) [reply docs/dns.md:10]: this merges to alpha, though.

Comment (sirdarckcat) [reply docs/dns.md:10]: ah thats what you mean..

Comment (sirdarckcat) [reply docs/dns.md:10]: lets fix this for alpha, we can fix beta independently

File: docs/dns.md (hunk 6 above)
Comment (sirdarckcat) [reply docs/dns.md:11]: k

File: docs/dns.md (hunk 7 above)
Comment (sirdarckcat) [reply docs/dns.md:8]: (this goes to alpha)

**Comment by sirdarckcat** [comment]
ptal - we need to fix the docs for beta significantly anyway

**Review by sroettger — APPROVED** [review]

**Pull request merged**